nss-day-cohort-45/hello-world-the-background-images
# Lrc jswork

## Changes 
adding js for famous cities, landmarks, and citizens to Ireland 

## Steps to test 
see that JS files are there for cities, landmarks, and citizens of Ireland
run code 
see that cities, landmarks, and citizens lists are generated
